src/DiagnosticCollection.spec.ts

```diff
@@ -9,7 +9,7 @@ import { expect } from 'chai';
 describe('DiagnosticCollection', () => {
     let collection: DiagnosticCollection;
     let diagnostics: BsDiagnostic[];
-    let workspaces: Workspace[];
+    let workspaces: Project[];
```

Comment:
Should you update workspaces to projects?

Reply:
Oops. Yep, good catch.